neo4j/_async/work/session.py

```diff
@@ -239,6 +245,126 @@ async def run(self, query, parameters=None, **kwargs):
 
         return self._auto_result
 
+    async def query(self, query, parameters=None, **kwargs):
+        """
+        Run a Cypher query within an managed transaction and
+        all the retries policy will be applied.
```

Comment:
The first paragraph of a doc string should be one line only.

If you want to did deeper https://peps.python.org/pep-0008/#documentation-strings and https://peps.python.org/pep-0257/